src/in-app-browser/in-app-browser.tsx

```diff
@@ -0,0 +1,40 @@
+import {StatusBar} from 'react-native';
+import InAppBrowser, {
+  InAppBrowserOptions,
+} from 'react-native-inappbrowser-reborn';
+import {notifyBugsnag} from '../utils/bugsnag-utils';
+
+export async function openInAppBrowser(
+  url: string,
+  dismissButtonStyle: 'cancel' | 'close' | 'done',
+  options?: InAppBrowserOptions,
```

Comment:
I really like having `dismissButtonStyle` as its own parameter, enabling users of this util function not needing to send in a full options object. I think I would prefer just removing the `options` param all together (YAGNI).